cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -120,11 +121,14 @@ func getCertsSubCommands(defaultKubernetesVersion string) []*cobra.Command {
 	}
 	addFlags(saCmd, &cfgPath, cfg, false)
 
-	subCmds := []*cobra.Command{allCmd, saCmd}
+	// "renew" command
+	renewCmd := certscmdphase.NewCmdCertsRenewal()
 
+	// add the rest of commands
 	certTree, err := certsphase.GetDefaultCertList().AsMap().CertTree()
 	kubeadmutil.CheckErr(err)
 
+	subCmds := []*cobra.Command{allCmd, saCmd, renewCmd}
```

Comment:
Personally it's more clear to move this line after L125. But it's ok as-is.

Reply:
i will move this:
```
certTree, err := certsphase.GetDefaultCertList().AsMap().CertTree()
kubeadmutil.CheckErr(err)
```
towards the beginning of the function instead,.

if it fails there is no point to attach the commands, even.

Reply:
actually this isn't good either.
will revert the change to reduce the noise.